routes/authRoutes.js

```diff
@@ -90,51 +92,123 @@ router.get('/dashboard', requireAuth, (req, res) => {
 		.populate('pendingConnectionRequests.requestedUser')
```

Comment:
At a quick glance, I think you're going to have a problem with the second `.populate(... requestingUser)` statement; as mongoose will only run the first `.populate()`.